source/stylesheets/base/_global-variables.scss

```diff
@@ -0,0 +1,30 @@
+// Breakpoints
+$small-breakpoint: 480px;
```

Comment:
One thing I like to do with sets of related variable names is to use a convention where it's what it is you're abstracting, then the variation. So, for your breakpoints, it'd be:

- `$breakpoint-small`
- `$breakpoint-medium`
- `$breakpoint-large`
- etc.

The idea behind this is it'll aid in discovery when using an editor that can perform code completion. So, I start to type "brea" and it'll show me all the breakpoint variable options I have to work with. Same for colors, font sizes, and the like.

Reply:
I'd plus 100 this point.

Reply:
Good to know - have we considered refactoring the thoughtbot.com site to follow this naming convention? (Could be a nice easy rainy Friday activity for me)

Reply:
That would be lovely... 😃 just be warned that there are a lot of varying stylesheets on thoughtbot.com and it might be more complicated than you imagine

Reply:
I'm talking a veeeeery rainy Friday :)